website/content/en/preview/getting-started/migrating-from-cas/scripts/step02-node-policies.sh

```diff
@@ -0,0 +1,11 @@
+aws iam attach-role-policy --role-name KarpenterInstanceNodeRole \
+    --policy-arn arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy
+
+aws iam attach-role-policy --role-name KarpenterInstanceNodeRole \
+    --policy-arn arn:aws:iam::aws:policy/AmazonEKS_CNI_Policy
+
+aws iam attach-role-policy --role-name KarpenterInstanceNodeRole \
+    --policy-arn arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly
+
+aws iam attach-role-policy --role-name KarpenterInstanceNodeRole \
+    --policy-arn arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore
```

Comment:
It appears because there is no line break at the end of the file, are we in 2022? 

```suggestion
    --policy-arn arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore

```